tests/example.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { test, expect } from '@playwright/test';
+
+const productCardByName = 'h5[data-test="product-name"]';
+
+test('Test 1: Verify login with valid credentials', async ({ page }) => {
+  await page.goto(process.env.WEB_URL + '/auth/login');
+  await page.locator('[data-test="email"]').fill( process.env.USER_EMAIL);
+  await page.locator('[data-test="password"]').fill(process.env.USER_PASSWORD);
+  await page.locator('[data-test="login-submit"]').click();
+  await expect(page).toHaveURL('https://practicesoftwaretesting.com/account');
+  await expect(page.locator('[data-test="page-title"]')).toContainText('My account');
+  await expect(page.locator('[data-test="nav-menu"]')).toContainText('Jane Doe');
```

Comment:
```suggestion
  await expect(page.locator('[data-test="nav-menu"]')).toContainText(process.env.USER_NAME);
```
Lets use env const since we have it.